django_comparison_dashboard/graphs.py

```diff
@@ -229,3 +247,321 @@ def sankey(data, filter_set: SankeyGraphFilterSet):
     "sankey": {"chart_function": sankey, "form_class": SankeyGraphFilterSet},
     "line": {"chart_function": line_plot, "form_class": LineGraphFilterSet},
 }
+# Define the color dictionary
+color_dict = {
```

Comment:
IMO this should be imported from JSON in settings.py (empty dict if file cannot be found) - so that it gets less hardcoded...